src/member.rs

```diff
@@ -392,13 +391,13 @@ impl Inner {
         let (tx, rx) = channel();
         let _ = thread::spawn(move || {
             #[cfg(not(test))]
-            let tick = periodic_ms(WAITING_TIME); // 2 minutes
+            let ticker = tick(Duration::from_millis(WAITING_TIME)); // 2 minutes
```

Comment:
Seems like [`after`](https://docs.rs/crossbeam-channel/0.4.2/crossbeam_channel/fn.after.html) would be a better choice here as you only wait for a single tick anyway?